src/SQLStore/Lookup/PropertyLabelSimilarityLookup.php

```diff
@@ -144,7 +144,7 @@ public function compareAndFindLabels( RequestOptions $requestOptions = null ) {
 		$similarities = $this->matchLabels( $propertyList, $withType );
 
 		usort( $similarities, function ( $a, $b ) {
-			return $a['similarity'] < $b['similarity'];
+			return $b['similarity'] <=> $a['similarity'];
```

Comment:
Patched in https://github.com/SemanticMediaWiki/SemanticMediaWiki/pull/5466/files